tests/components/meteo_france/conftest.py

```diff
@@ -0,0 +1,12 @@
+"""Meteo-France generic test utils."""
+from unittest.mock import patch
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def patch_requests():
+    """Stub out services that makes requests."""
+    with patch("homeassistant.components.meteo_france.meteofranceClient"):
+        with patch("homeassistant.components.meteo_france.VigilanceMeteoFranceProxy"):
```

Comment:
A single `with` statement is enough for both patches.

```py
with patch(...), patch(...):
    yield
```
If it's hard to fit both patches on the lines readable, after black, we can save them in variables first.

```py
patch_client = patch(...)
patch_proxy = patch(...)

with patch_client, patch_proxy:
    yield
```